packages/babel-parser/test/fixtures/es2015/let/let-with-linebreak-and-escaped/input.js

```diff
@@ -0,0 +1,2 @@
+while (0) let
+\u0061 = 1;
```

Comment:
Currently Babel parses it as a `WhileStatement` including `let` variable declaration. This is incorrect as it should share the AST structure with
```js
while (0) let
a = 1;
```
which consists of a WhileStatement and an ExpressionStatement.